Resources/Prototypes/_NF/Entities/Objects/Fun/spaceblade.yml

```
@@ -0,0 +1,274 @@
+- type: entity
+  id: SpaceBlade
+  parent: AirTank
+  name: space blade
+  description: Let it drip
+  abstract: true
+  components:
+  - type: GasTank
+    outputPressure: 0.1
+    maxOutputPressure: 0.1
+    valveOutputRate: 50
+    air:
+      volume: 1
+      moles:
+         - 0.4101137219 # oxygen
+      temperature: 293.15
+  - type: Physics
+    bodyType: Dynamic
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.25,-0.25,0.25,0.25"
+        density: 0.1
+        mask:
+        - InteractImpassable
+        layer:
+        - InteractImpassable
+  - type: Item
+    size: Small
+    sprite: _NF/Objects/Fun/spaceblade.rsi
+  - type: Sprite
+    sprite: _NF/Objects/Fun/spaceblade.rsi
+    state: redorange
+    scale: 0.5, 0.5
+  - type: EmitSoundOnCollide
+    sound:
+      path: /Audio/Items/desk_bell_ring.ogg
+      params:
+        variation: 0.65
+        volume: -5
+  - type: MeleeWeapon
+    damage:
+      types:
+        Slash: 5
+
+- type: entity
+  id: SpaceBladeRedOrange
+  parent: SpaceBlade
+
+- type: entity
+  id: SpaceBladeBlue
+  parent: SpaceBlade
+  components:
+  - type: Sprite
+    state: blue
+
+- type: entity
+  id: SpaceBladeFrontier
+  parent: SpaceBlade
+  components:
+  - type: Sprite
+    state: frontier
+
+- type: entity
+  id: SpaceBladeGoldBlue
+  parent: SpaceBlade
+  components:
+  - type: Sprite
+    state: goldblue
+
+- type: entity
+  id: SpaceBladeMail
+  parent: SpaceBlade
+  components:
+  - type: Sprite
+    state: mail
+
+- type: entity
+  id: SpaceBladeEyeball
+  parent: SpaceBlade
+  components:
+  - type: Sprite
+    state: eyeball
+
+- type: entity
+  id: SpaceBladeBlade
+  parent: SpaceBlade
+  components:
+  - type: Sprite
+    state: blade
+
+- type: entity
+  id: SpaceBladeCheck
+  parent: SpaceBlade
+  components:
+  - type: Sprite
+    state: check
+
+- type: entity
+  id: SpaceBladeRune
+  parent: SpaceBlade
+  description: Birthday parties are yearly reminders that as soon as you're born you start dying, unless you let it rip
+  components:
+  - type: Sprite
+    state: rune
+
+- type: entity
+  id: SpaceBladeMc
+  parent: SpaceBlade
+  description: Let it McDrip
+  components:
+  - type: Sprite
+    state: mcblade
+  - type: EmitSoundOnUse
+    sound:
+      path: /Audio/Effects/sizzle.ogg
+      params:
+        playOffsetSeconds: 3
```

Increase the grease.
```suggestion
      path: /Audio/Nyanotrasen/Machines/deepfryer_basket_add_item.ogg
```